types/thumbsupply/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for thumbsupply 0.4
+// Project: https://github.com/RyanAfrish7/thumbsupply#readme
+// Definitions by: MagiCzOOz <https://github.com/MagiCzOOz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    forceCreate?: boolean;
+    size?: Size;
+    timestamp?: string;
+    cacheDir?: string;
+    mimetype?: string;
+}
+
+interface Size {
+    name?: string;
+    width: number;
+    height: number;
+}
+
+interface AvailableSize {
+    MEDIUM: Size;
+    LARGE: Size;
+}
+
+declare class ThumbSupply {
+    _thumbSuppliers: ThumbnailSupplier;
+    _defaultOptions: Options;
+    ThumbSize: AvailableSize;
+
+    constructor();
+    _registerThumbSupplier(mimetype: string, Thumbsupplier: ThumbnailSupplier): void;
+    _fetchThumbnailSupplier(file: string, options?: Options): ThumbnailSupplier;
+    generateThumbnail(file: string, options?: Options): Promise<string>;
+    lookupThumbnail(file: string, options?: Options): Promise<string>;
+}
+
+declare class ThumbnailSupplier {
+    filetype(): string;
+    hashFile(file: string): string;
+    getThumbnailFileName(file: string): string;
+    constructor(options: Options);
+    createThumbnail(file: string): void;
+    getThumbnailLocation(file: string): string;
+}
+
+declare const thumbsupply: ThumbSupply;
+
+export default thumbsupply;
```

Comment:
I believe you should use `export = thumbsupply` instead of `export default thumbsupply` here, since the original package implementation exports this as `module.exports = new ThumbSupply()`.

Reply:
You are absolutely right. Thank you for your review. I commit the change right now.